server/test-utils/db-helpers.ts

```diff
@@ -2,10 +2,20 @@ import dotenv from 'dotenv';
 import mongoose from 'mongoose';
 import UserModel from '../entities/User';
 import { fakeUser, fakeUser2 } from './fake-user';
+import { MongoMemoryServer } from 'mongodb-memory-server-core';
+import isCI from 'is-ci';
+
 dotenv.config();
 
+const mongod = new MongoMemoryServer();
+
 export const connect = async () => {
-  const uri = process.env.TEST_DB_URI
+  let uri = process.env.TEST_DB_URI;
+
+  if (isCI) {
+    uri = await mongod.getConnectionString();
+  }
+
```

Comment:
Cool trick huh? 

When in CI env use mongo memory server & on local development env use local mongodb database as usual.